README.md

```diff
@@ -94,114 +94,130 @@ Snap deploys as a binary, which makes requirements quite simple. We've tested on
+$ cd ~/snap/plugins/
+$ snapctl plugin load snap-collector-mock1
+$ snapctl plugin load snap-processor-passthru
+$ snapctl plugin load snap-publisher-file
 ```
 
-Let's look at what plugins we have loaded now:
-
+Let's look at what plugins you have loaded now:
 ```
-$ <snapBinPath>/snapctl plugin list
+$ snapctl plugin list
 NAME             VERSION         TYPE            SIGNED          STATUS          LOADED TIME
 mock             1               collector       false           loaded          Tue, 17 Nov 2015 14:08:17 PST
 passthru         1               processor       false           loaded          Tue, 17 Nov 2015 14:16:12 PST
 file             3               publisher       false           loaded          Tue, 17 Nov 2015 14:16:19 PST
 ```
-### Running Tasks
 
-Tasks can be in JSON or YAML format. Let's start one of the [example tasks](./examples/tasks/mock-file.yaml) from the `examples/tasks/` directory:
+You now have one of each plugin type loaded into the framework. To begin collecting data, you need to create a task.
+
+### Running Tasks
+Tasks are most often shared as a Task Manifest and is written in JSON or YAML format. Make a copy of [this example task](./examples/tasks/mock-file.yaml) from the `examples/tasks/` directory on your local system and then start the task:
 
 ```
-$ cd <snapDirectoryPath>
-$ <snapBinPath>/snapctl task create -t examples/tasks/mock-file.yaml
+$ cd ~/snap
+$ curl https://raw.githubusercontent.com/intelsdi-x/snap/master/examples/tasks/mock-file.yaml > mock-file.yaml
+$ snapctl task create -t mock-file.yaml
 Using task manifest to create task
 Task created
 ID: 8b9babad-b3bc-4a16-9e06-1f35664a7679
 Name: Task-8b9babad-b3bc-4a16-9e06-1f35664a7679
 State: Running
 ```
 
-From here, you should be able to do 2 things:
+This task generates mock data, "processes" it through a passthru mechanism and then publishes it to a file. Now with a running task, you should be able to do 2 things:
 
 See the data that is being published to the file:
 ```
 $ tail -f /tmp/snap_published_mock_file.log
+^C
 ```
 
-Or actually tap into the data that snap is collecting using the task ID to watch the task:
+Or tap into the data that Snap is collecting using the Task ID to watch the task (note: your Task ID will be different):
 ```
-$ <snapBinPath>/snapctl task watch 8b9babad-b3bc-4a16-9e06-1f35664a7679
+$ snapctl task watch 8b9babad-b3bc-4a16-9e06-1f35664a7679
 ```
 
-If the task ID already scrolled by, you can get the task ID with:
+If the Task ID already scrolled by, you can list it with:
 ```
-$ <snapBinPath>/snapctl task list
+$ snapctl task list
 ```
 
+Nice work - you're all done with this example. You can continue to run more tasks using the loaded plugins (why not create a new Task Manifest that publishes mock data to another file?). Once you're done familiarizing yourself with the task creation process, stop any running tasks and unload any plugins you no longer wish to use:
```

Comment:
@tiffanyfj @thomastaylor312 what do you recommend here? I'm trying to think through the final "we're done here" comments